docs/pages/database-access/guides/rds.mdx

```diff
@@ -20,268 +20,40 @@ This guide will help you to:
 - AWS account with RDS and Aurora databases and permissions to create and attach
   IAM policies.
 
-## Step 1/6. Install Teleport
+## Step 1/7. Install Teleport
 
 (!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-## Step 2/6. Create Teleport user
+## Step 2/7. Create Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 3/6. Configure IAM
+## Step 3/7. Create database agent configuration
 
-### Create IAM policy for Teleport
-
-Teleport needs AWS IAM permissions to be able to:
-
-- Discover and register RDS instances and Aurora clusters.
-- Configure IAM authentication for them.
-
-Go to the [Policies](https://console.aws.amazon.com/iamv2/home#/policies) page
-and create a managed IAM policy for the database agent.
-
-The exact set of required permissions depends on whether you're connecting to
-RDS instances or Aurora clusters (or both), as well as the IAM identity your
-Teleport database agent will be using (user or role).
-
-<Tabs>
-  <TabItem label="RDS / IAM user">
-  Use this policy if you're connecting to RDS instances and your Teleport
-  database agent runs as IAM user (for example, uses AWS credentials file).
-  ```json
-  {
```

Comment:
I think it might be useful to have these examples of policies required for different deployment scenarios (RDS, Redshift) somewhere in the documentation. We have this page: https://goteleport.com/docs/database-access/reference/aws/. What do you think about moving these (and Redshift) example policies there?

Reply:
Makes sense. I added a section on the reference page about the auto-discovery policies.